### ISSUE TITLE
minor: use full object for argv

### ISSUE THREAD

**Issue by warner** [Agoric/cosmic-swingset#125]
The bootstrap vat gets an argument named `argv` from the host. We currently use this to tell the swingset machine what mode it should use, along with information about the addresses of remote machines to which it should expect to connect.

I named this `argv` because that's how unix systems pass command-line arguments into the process, and so much of SwingSet looks like a kernel talking to a userspace process. But we don't have to be limited by vectors of arguments here: the kernel does a full `JSON.parse()` of whatever gets passed in. So it might be cleaner to use named arguments than merely positional ones.

I don't know if that means we should change the name of this argument (maybe to `arguments`, but that collides with some JavaScript things). How about we just declare that the `v` in `argv` stands for "values"?

**Comment by erights** [comment]
Do *not* use `arguments`. It is bad JS magic you don't want here.

If it were exactly an array of strings that could be imagined to have come from a command line, even if it doesn't, `argv` would be good. Otherwise, something else.

**Comment by warner** [comment]
Ok let's use `args`.

**Comment by warner** [comment]
Of course, it's a positional argument, so the `bootstrap.js` code that receives it can name it any way it likes. This will just change the kernel internals and the conventions for the bootstrap code.

**Comment by warner** [comment]
in the old repo. this was cosmic-swingset issue 125
